src/configLoaders/DefaultJestConfigLoader.ts

```diff
@@ -17,7 +17,7 @@ export default class DefaultJestConfigLoader implements JestConfigLoader {
   }
 
   public loadConfig(): JestConfiguration {
-    const jestConfig = this.readConfigFromJestConfigFile() || this.readConfigFromPackageJson();
+    const jestConfig = this.readConfigFromJestConfigFile() || this.readConfigFromPackageJson() || {};
 
     if (!jestConfig) {
       throw new Error('Could not read Jest configuration, please provide a jest.config.js file or a jest config in your package.json');
```

Comment:
This can be removed right? This error cannot occur?

Reply:
You're definitely right! Removed these lines...